src/server/status/lib/metrics.js

```diff
@@ -98,8 +104,11 @@ export class Metrics {
         avg_in_millis: isNaN(avgInMillis) ? undefined : avgInMillis, // convert NaN to undefined
         max_in_millis: maxInMillis
       },
-      requests: keysToSnakeCaseShallow(get(hapiEvent, ['requests', port])),
-      concurrent_connections: get(hapiEvent, ['concurrents', port]),
+      requests: {
+        ...requestDefaults,
+        ...keysToSnakeCaseShallow(get(hapiEvent, ['requests', port]))
+      },
+      concurrent_connections: hapiEvent.concurrent_connections
```

Comment:
Something seems to be up with capturing requests stats from ops event data. I'm not seeing the requests stats ever count anything
![image](https://user-images.githubusercontent.com/908371/44550614-9f911680-a6d9-11e8-8cc3-2a309040b34b.png)

![image](https://user-images.githubusercontent.com/908371/44550698-d9fab380-a6d9-11e8-870d-c580628e7562.png)

Reply:
Thanks for taking a looking and helping me catch this. I'll ping you when I have a fix.

Reply:
This was a fun one. Turns out the library the official hapi library for getting this data (hapijs/oppsy) emits events with objects that it then later resets. This was just a race condition with also getting the number of connections on the server. I've changed this to do a deep copy of the event data, but I'm also going to submit a patch upstream.